src/app/personside/MainLayout.tsx

```diff
@@ -7,51 +7,94 @@ import Lameller from './lameller/Lameller';
 const Wrapper = styled.div`
         width: 100%;
         flex-grow: 1;
-        @media (${props => props.theme.media.wideScreen}) {
-          display: flex;
-          flex-flow: row nowrap;
-        }
-        padding: ${props => props.theme.margin.layout};
         overflow-y: scroll;
     `;
 
 const PersonOversiktsPanel = styled.section`
-        margin-bottom: ${props => props.theme.margin.layout};
-        @media (${props => props.theme.media.wideScreen}) {
-          flex: 1 0.1 60%;
-          margin-right: ${props => props.theme.margin.layout};
-        }
+        width: 65%;
+        padding: ${props => props.theme.margin.layout};
         > * {
           box-shadow: ${props => props.theme.boxShadow.layout};
+        }
+        > *:not(:last-child){
           margin-bottom: ${props => props.theme.margin.layout};
         }
     `;
 
-const DialogPanel = styled.section`
-        min-height: 700px;
+interface DialogPanelProps {
+    wrapperHeight: number;
+    wrapperWidth: number;
+    absoluteTop: number;
+}
+
+const DialogPanel = styled<DialogPanelProps, 'section'>('section')`
+        position: fixed;
+        left: ${props => props.wrapperWidth * 0.65}px;
+        top: ${props => props.absoluteTop}px;
+        width: ${props => props.wrapperWidth * 0.35}px;
+        height: ${props => props.wrapperHeight}px;
+        padding: ${props => props.theme.margin.layout};
+        padding-left: 0;
         display: flex;
         flex-flow: column nowrap;
-        @media(${props => props.theme.media.wideScreen}) {
-          flex: 1 0.1 20%;
-        }        
         > * {
           box-shadow: ${props => props.theme.boxShadow.layout};
           background-color: white;
         }
     `;
 
-function MainLayout() {
-    return (
-        <Wrapper>
-            <PersonOversiktsPanel>
-                <VisittkortContainer/>
-                <Lameller/>
-            </PersonOversiktsPanel>
-            <DialogPanel>
-                <ComponentPlaceholder height={'100%'} name={'Dialog Panel'}/>
-            </DialogPanel>
-        </Wrapper>
-    );
+interface MainLayoutState {
+    wrapperWidth: number;
+    wrapperHeight: number;
+    wrapperAbsoluteTop: number;
+}
+
+class MainLayout extends React.Component<{}, MainLayoutState> {
+
+    private wrapperRef: HTMLDivElement;
+
+    constructor(props: {}) {
+        super(props);
+        this.state = { wrapperWidth:  0, wrapperHeight: 0, wrapperAbsoluteTop: 0};
+    }
+
+    componentDidMount() {
+        this.findWrapperDimensions();
+        window.addEventListener('resize', () => this.findWrapperDimensions());
```

Comment:
Kan det taes en runde på det som skjer her? Jeg er ikke helt sikker på hvordan skissene er for nye-modia. 
Men etter det jeg kan forstå så ser det ut som det reimplementeres en del funksjonalitet som css støtter ut-av-boksen. 

Selvom det kanskje er enkelt nå som man har styles-components så er jeg veldig usikker på det faktisk er en god retning å bevege seg i ettersom dette kommer til å bli en relativt stor løsning, og jo mer "hjelp" vi kan få fra browseren jo bedre.

Reply:
det kan vi!

Som du kanksje har sett er ikke dette ferdig kode, men et av tre forslag til hvordan layouten kan løses. Tenkte i første omgang å ta med dette til designerne for å få tilbakemelding på hva de ønsker seg i den endelige løsningen.

og bare hyggelig om du vil lage noen løsningsforslag til dette @nutgaard, klart vi ønsker en løsning med ren css

Reply:
@nutgaard: Problemet er at vi trenger noe som fungerer som `position: sticky` men som fungerer i IE11.